plc4j/drivers/opcua/src/main/java/org/apache/plc4x/java/opcua/protocol/OpcuaProtocolLogic.java

```diff
@@ -195,7 +195,7 @@ public CompletableFuture<PlcReadResponse> read(PlcReadRequest readRequest) {
                         future.complete(new DefaultPlcReadResponse(request, status));
                         return;
                     }
-                } catch (ParseException e) {
+                } catch (Exception e) {
```

Comment:
What was the reasoning behind this?

Reply:
The uncaught PlcUnsupportedDataTypeException from PlcValueHandler caused the CompletableFuture not to complete, and the thread is hanging on CompletableFuture.get().